packages/nestjs-query/src/dataProvider/index.ts

```diff
@@ -472,7 +480,11 @@ const dataProvider = (client: GraphQLClient): Required<DataProvider> => {
             variables = gqlMutation.variables;
           }
 
-          const response = await client.request<BaseRecord>(query, variables);
+          const response = await _client.request<BaseRecord>({
+            document: query,
+            variables,
+            requestHeaders: headers,
```

Comment:
Oops, those aren't needed
```suggestion
          requestHeaders: headers,
```